src/styles/partials/template/_sidebar.scss

```diff
@@ -10,15 +12,23 @@
   p {
     font-family: $default-heading-font;
     font-size: $smallest-heading-font-size;
+    margin: $default-margin 0;
   }
 
   h2 {
     border: none;
     letter-spacing: 10px;
     font-weight: $font-bold;
+    font-size: $h3-font-size;
   }
 
   p {
     line-height: 30px;
   }
 }
+
+@media screen and (max-width: $medium-breakpoint) {
+  .dg_sidebar:first-of-type {
```

Comment:
Wondering if the `first-of-type` selector is necessary. In theory, there should only be one dg_sidebar per page, correct? Thinking this should simply be .`dg_sidebar`.

Reply:
We added this for in case of adding multiple side bars. Although this is probably true that only one side bar will exist per page the XD document did show a possibility of having more then one such as Contact Us and Follow Us.

Reply:
@tmccoy529 I thought we had talked about keeping it consistent with add margin to the bottom of things? Would be something like

```scss
.dg_sidebar {
  margin-bottom: $margin-larger;

  &:last-child {
    margin-bottom: 0;
  }
}
```

If you inspect any of the typography, you'll notice the top margin has been removed but a bottom margin exists. Probably something that I could have brought up for discussion. 

My argument is just to keep it consistent, if `top` is the way to go, then I'm fine, we should just switch the other items.

Reply:
@tmccoy529 after going through this pr from the bottom up, I realize what you are doing now. My comment may be valid and may not be depending on what happens with the other comments. I'll wait a bit until the other comments are addressed to chime back in on this particular conversation.

Reply:
Per our discussion, dg_main_content needs to have a bottom margin added thereby avoiding the media query entirely for side bar but side bar also needs a bottom of 40 for stacking. This has been done.